Dockerfile.arm64

```diff
@@ -0,0 +1,23 @@
+FROM alpine as nim
+LABEL maintainer="setenforce@protonmail.com"
+
+RUN apk --no-cache add gcc git libc-dev libsass-dev "nim=1.6.10" nimble pcre
```

Comment:
It's `nim=1.6.8-r0` and not `nim=1.6.10`

Reply:
Resolved.